dev/com.ibm.ws.ejbcontainer.timer/bnd.bnd

```diff
@@ -30,8 +30,11 @@ Include-Resource: \
 	com.ibm.ws.context;version=latest,\
 	com.ibm.ws.ejbcontainer;version=latest,\
 	com.ibm.ws.ejbcontainer.core;version=latest,\
-	com.ibm.ws.kernel.service,\
+	com.ibm.ws.kernel.service;version=latest,\
 	com.ibm.ws.logging,\
+	com.ibm.ws.container.service;version=latest,\
 	com.ibm.websphere.org.osgi.core,\
 	com.ibm.websphere.org.osgi.service.component,\
-	com.ibm.wsspi.org.osgi.service.component.annotations
+	com.ibm.wsspi.org.osgi.service.component.annotations,\
```

Comment:
indent is wrong here.... not sure if it is a tabs/blanks issue.... but this last line should line up.

Also... last line should NOT end with ,\   I think that could cause problems in the future if anything else were ever added after this that is not part of buildpath.

Reply:
Made change